portfolio/src/main/java/com/google/sps/servlets/Log.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.User;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/log")
+public class Log extends HttpServlet {
+
+  /** Write true into the server iff the user is logged in. */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+
+    response.setContentType("text/html");
+
+    if (userService.isUserLoggedIn()) {
+      response.getWriter().print("true");
+    } else {
+      response.getWriter().print("false");
+    }
```

Comment:
nit: If this is just to set the print value to "true" or "false", then you might want to use a conditional operator.

```suggestion
    String isLoggedIn = userService.isUserLoggedIn() ? "true" : "false";
    response.getWriter().print(isLoggedIn);
```